src/tribler/core/utilities/tests/test_utilities.py

```diff
@@ -187,6 +187,24 @@ def test_parse_query():
     assert actual == expected
 
 
+def test_parse_bool():
+    assert not parse_bool('')
+    assert not parse_bool('false')
+    assert not parse_bool('False')
+    assert not parse_bool('0')
+    assert not parse_bool('0.0')
+    assert not parse_bool('0.0e2')
```

Comment:
Is it necessary to support float numbers here? What is the use case for that?

Reply:
it was mostly to be conformant with what bool() would do for a float value.
But i will remove it.